docs/modules/commons/pages/vividus-steps.adoc

```diff
@@ -445,7 +445,25 @@ Examples:
 file:///${examples-table-temporary-file}
 ----
 
+=== File Existence Validation
```

Comment:
is there any reason to start each word from upper-case letter?

Reply:
no reason )